### ISSUE TITLE
Impex Code Style Formatter

### ISSUE THREAD

**Issue by cdigruttola** [epam/sap-commerce-intellij-idea-plugin#540]
Good morning,

I'm using the 2023.2.5 version with IntelliJ 2023.2.
I'm having problems with ImpEx formatter. 
As you can see in the screen, I cannot set spaces before and after semicolon. Furthermore, it seems to ignoring my formatter, because for the Impex part I have these settings, but when I'm going to format, the spaces are added.
```
<ImpexCodeStyleSettings>
    <option name="SPACE_AFTER_FIELD_VALUE_SEPARATOR" value="false" />
    <option name="SPACE_BEFORE_FIELD_VALUE_SEPARATOR" value="false" />
    <option name="SPACE_AFTER_PARAMETERS_SEPARATOR" value="false" />
    <option name="SPACE_AFTER_COMMA" value="false" />
    <option name="SPACE_AFTER_ATTRIBUTE_SEPARATOR" value="false" />
    <option name="SPACE_AFTER_FIELD_LIST_ITEM_SEPARATOR" value="false" />
    <option name="SPACE_AFTER_ASSIGN_VALUE" value="false" />
    <option name="SPACE_BEFORE_ASSIGN_VALUE" value="false" />
    <option name="SPACE_AFTER_ALTERNATIVE_PATTERN" value="false" />
    <option name="SPACE_BEFORE_ALTERNATIVE_PATTERN" value="false" />
  </ImpexCodeStyleSettings>

```

![Screenshot 2023-08-07 at 11 34 06](https://github.com/epam/sap-commerce-intellij-idea-plugin/assets/9153905/8861635d-3c09-49cd-bf88-1109bfa96b4f)

**Comment by mlytvyn** [comment]
@cdigruttola thank you very much for reporting this, I’ll look into it and include fix in the next release

**Comment by mlytvyn** [comment]
@cdigruttola , thank you, fixed in scope of the #541  and will be available in the next `2023.2.6` release
